backend/src/main/java/saci/application/LevelController.java

```diff
@@ -0,0 +1,67 @@
+package saci.application;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import saci.domain.model.Level;
+import saci.domain.service.LevelService;
+
+@RestController
+@RequestMapping("/api/levels")
+@RequiredArgsConstructor
+public class LevelController {
+
+    private final LevelService levelService;
+
+    @Operation(summary = "Create a Level")
+    @ApiResponses(
+            value = {
+                @ApiResponse(
+                        responseCode = "201",
+                        description = "The Level was created",
+                        content = {
+                            @Content(
+                                    mediaType = "application/json",
+                                    schema = @Schema(implementation = Level.class))
+                        })
+            })
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public Level createLevel(@RequestBody Level level) {
+        return levelService.createLevel(level);
+    }
+
+    @Operation(summary = "Get all of the Levels")
+    @ApiResponses(
+            value = {
+                @ApiResponse(
+                        responseCode = "200",
+                        description = "List of all Levels",
+                        content = {
+                            @Content(
+                                    mediaType = "application/json",
+                                    array =
+                                            @ArraySchema(
+                                                    schema = @Schema(implementation = Level.class)))
+                        })
+            })
+    @GetMapping
+    public ResponseEntity<List<Level>> getLevels() {
```

Comment:
Your task is about level creation, why do you have an endpoint to get levels? Can you remove ALL code not related to your task?